client/src/components/mapping/FhirMappingPanel/AddResource/index.tsx

```diff
@@ -106,18 +103,19 @@ const AddResource = ({ callback }: Props) => {
     });
 
     // Create the new resource
-    const {
-      data: { createResource: createdResource }
-    } = await createResource({
+    const responseCreateResource = await createResource({
```

Comment:
you can destructure `data` here, it will still exist even if an error is returned i think

Reply:
I tried it and I think the whole mutation returns undefined if an error is caught in the middle (and the onError function is called)